### ISSUE TITLE
Publish news updates

### ISSUE THREAD

**Issue by asimonw** [emfoundation/ce100-app#30]
As an EMF admin, I can publish and edit simple platform and programme news items, so that I can keep members up to date about new platform features and important programme news.

**Comment by JMurphyWeb** [comment]
@asimonw where would you like the admin to be redirected to on save, cancel and archive?

Additionally, is archive supposed to be here, or is that covering the 'edit news' view?

**Comment by asimonw** [comment]
@JMurphyWeb Please redirect users to the News listing page.

The archive functionality is really better thought of as a disable/enable (from an admin point of view, at least). This means that an admin should be able to create a post, but not "publish" immediately, which makes the "Archive" feature relevant already on Create.

**Comment by JMurphyWeb** [comment]
So it sounds like the archive functionality is more like 'Save, without publishing'.

**Comment by asimonw** [comment]
@JMurphyWeb Yes, we made this point before about orgs and people as well, but I think it might have got lost in translation. 

This is perhaps not the best place to write this, but for us there is a difference between Archive/Unarchive and Enable/Disable. The former is more like a soft delete for end users, while the latter is an admin functionality, which might be more often used to create content before "activating" (or publishing) it on the app. This implies the following:
* Orgs --> enable/disable, admin
* People --> enable/disable, admin
* Insights/Resources --> enable/disable, admin
* Challenges --> archive/unarchive, primary users

Technically they are very similar, hence the possible confusion. We've noticed that you always use the words archive/unarchive in the app, but ideally, we'd use the words enable/disable for the admin functionality. 

Another reason why there's a difference:
* Disabled: item is not visible to primary users, but it is to admins, for whom it appears in the list with a red dot.
* Archived: item is visible to primary users, but it appears in a separate list (e.g. archived challenges), so that it's out of the user's way. It also doesn't appear in the list for admins. If the user decided to archive it, it's not content that should be easily accessible to admins.

I hope that this makes the distinction clearer. Happy to copy this comment to a more appropriate place.

**Comment by JMurphyWeb** [comment]
It's really good to have the difference laid out, I had not realised there was this clear distinction being made. 

@mk4111 I think this is good to read. I am not sure where we should put it (possibly readme?)

**Comment by mk4111** [comment]
@JMurphyWeb - I agree, I think README.md would be good!

**Comment by JMurphyWeb** [comment]
@asimonw you suggested having a checkbox for the enabled/disabled part of this form. I think it makes a lot of sense to implement that here. Would you like the default state to be `enabled`, but leaving the admin the option to `disable news item` by checking the checkbox.

**Comment by asimonw** [comment]
@JMurphyWeb You're right, this shouldn't say "Archive" here, but Enable/Disable as you suggest.

The default state should indeed be Enabled, but I think it makes more sense to have the checkbox be called "Enabled" and check it be default. That means that checked means "active" and unchecked means "non-active", which is a more natural mental model.
